index.html

```diff
@@ -1916,9 +1916,12 @@ <h4 id="maturity-levels">6.1.2 Maturity Levels</h4>
               corresponds to the "Last Call Working Draft" discussed in the Patent Policy.</li>
           </ul>
         </dd>
-        <dd><strong>Note</strong>: Candidate Recommendations are expected to be acceptable as Recommendations. Announcement of a
-          different next step <em class="rfc2119">should</em> include the reasons why the change in expectations comes at so late
-          a stage.</dd>
+        <dd><strong>Note</strong>: Advancing to Candidate Recommendation indicates that
+          no further improvement is expected without additional implementation experience and testing.
```

Comment:
No further _substantive_ improvement?

WGs typically re-read their specs and make editorial improvements, and that should fall within the range of what is expected.

Reply:
Right, I was wondering about something like this.

The flip side is that just because editorial improvements are allowed doesn't mean that they are expected, and since we expect CRs to be acceptable as RECs, we should mostly not enter CR when things are unreadable and need massive editorial improvements.

Then again, if we adopt the suggestion you propose, combining it with the rest of the text, it means that we're supposed to enter CR with high quality prose that would be acceptable as a REC, but that this is not incompatible with having further plans for editorial improvements, which sounds reasonable to me.

I'll ponder that for a short white, but unless someone pushes back, I'll probably make this change.

Reply:
I think the key word here is *expected*. No, we don't want CRs that have known editorial issues either. On the other hand, this sentence is clearly about the technical content. No technical improvement is expected without...?